backend/qemu.pm

```diff
@@ -874,6 +874,13 @@ sub start_qemu {
         for (my $i = 0; $i < $num_networks; $i++) {
             if ($vars->{NICTYPE} eq "user") {
                 my $nictype_user_options = $vars->{NICTYPE_USER_OPTIONS} ? ',' . $vars->{NICTYPE_USER_OPTIONS} : '';
+                if ($vars->{QEMU_ENABLE_SMBD}) {
+                    my $smbd_bin = find_bin('/usr/sbin/', 'smbd');
+                    $smbd_bin                             or die 'No server to provide SMB/CIFS services to SUT';
+                    (qx($smbd_bin --version) =~ /(\d.*)/) or warn 'smbd version regex was not matched';
+                    bmwqemu::diag "smbd version found on worker host: $1";
```

Comment:
why is the version important? Do you rely on certain features?

Reply:
In the past there has been a reported incompatibility.
https://unix.stackexchange.com/questions/165554/shared-folder-between-qemu-windows-guest-and-linux-host

Reply:
I see. To simplify I think we can ignore that assuming that we have newer versions of qemu. The referenced issue was referring to needing at least qemu >= 2.2.0 so maybe instead of smbd check qemu and die instead of warn+diag? I always favor aborting all operations in case of unforeseen cases. If we find out we need to gracefully handle that we can still change it later on. But all that is uncritical when the feature is only enabled anyway based on the feature switch you introduce